tools/fog-local-network/README.md

```diff
@@ -120,7 +120,6 @@ Usage
 1. Activate the virtualenv: `. ./env/bin/activate`
 1. Install requirements: `pip install --upgrade pip && pip install -r requirements.txt`
 1. Compile the protobuf files into a python module: `./build.sh`
-1. Start the sample paykit remote wallet server: `cargo run --bin sample_paykit_remote_wallet`
 1. Start the fog conformance test script: `./tools/fog_conformance_test.py`
```

Comment:
path is in `./tools/fog-local-network/fog_conformance_test.py` ?

Reply:
Sorry, I don't understand the question. This is no longer needed since `./tools/fog-local-network/fog_conformance_test.py` takes care of running this (https://github.com/mobilecoinfoundation/fog/pull/49/files#diff-9f39fb15f8a45eb7f0ba19754a9e3ab1d1a05d65e676a2d79b6d0d228489f7fbR365)

Reply:
oh wow, yea that question got all kinds of butchered. sorry bout that! I meant:
"Is `./tools/fog_conformance_test.py` the same path as `./tools/fog-local-network/fog_conformance_test.py` "? 

(B/c I didn't see a path to `./tools/fog_conformance_test.py`)

Reply:
Oh yeah, that is a typo. I will fix it. Thanks!